CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 #### 1.9.8-SNAPSHOT
 
+- **[FEATURE]** (`tx-graphics`) Add new graphics module which add graphics utilities extension methods.
```

Comment:
`tx-graphics` -> `ktx-graphics`

Reply:
The very first changelogs do not contain that, but currently when we're adding a new module, we tend to list all added features in a sublist.

```markdown
* **[FEATURE]** (`ktx-graphics`) Add new graphics module which add graphics utilities extension methods.
  * Added `xyz` extension method.
  (...)
```